solidity/random-beacon/deploy/06_authorize_random_beacon.ts

```diff
@@ -0,0 +1,22 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
+  const { getNamedAccounts, deployments } = hre
+  const { deployer } = await getNamedAccounts()
+  const { execute } = deployments
+
+  const RandomBeacon = await deployments.get("RandomBeacon")
+
+  await execute(
+    "ReimbursementPool",
+    { from: deployer },
+    "authorize",
+    RandomBeacon.address
+  )
+}
+
+export default func
+
+func.tags = ["RandomBeaconAuthorize"]
+func.dependencies = ["ReimbursementPool", "RandomBeacon"]
```

Comment:
`05_approve_random_beacon.ts` and `06_authorize_random_beacon.ts` names are not explanatory enough. We could swap their content and the names would still be pretty aligned with what's happening.

Can we rename to, for example, `05_approve_random_beacon_in_token_staking.ts` and `06_authorize_random_beacon_in_reimbursement_pool.ts` ?